client/index.js

```diff
@@ -0,0 +1 @@
+console.log('Hi from client index.js');
```

Comment:
Was just to confirm that my frontend bundling worked as intended. Have taken note of it.